### DISCUSSION TITLE
Sync client Keeshare DBs automatically when

### DISCUSSION THREAD

**Discussion by yi0n** [keepassxreboot/keepassxc#7461] [Q&A]
Hi,

maybe I just overlooked something, but I have a master db, from where I export some groups through KeeShare.
personA.kdbx.share (export/import)
personB.kdbx.share (export/import)
sharedwebsites.kdbx.share (only export)

When I change a password in sharedwebsites.kdbx.share, person A and B have the changes only **after the next start** of their personal personA.kdbx, where their DB gets reopened and therefore the imported sharedwebsites.kdbx.share reloaded.

Is there a way to force update or automatically watch for changes so that the "clients" get the changes immediately?

Thanks!

**Comment by droidmonkey** [comment]
Make sure everyone is running version 2.6.6.

**Reply by yi0n** [reply]
Everyone is running 2.6.6. Application is started from a network drive with a .bat script:
_start F:\SharedApps\KPXC\KeePassXC.exe --keyfile X:\pwmgmnt\keys\\%username%.keyx X:\pwmgmnt\\%username%.kdbx_

**Reply by yi0n** [reply]
I observed the syncing behavior for some days know and I can tell that an update to the "main" database file, from which client databases import a shared group, won't get the change immediately. Only after restart. Is this to be expected?

**Reply by droidmonkey** [reply]
Changes are not saved until the parent database is saved.
